addons/sms/models/sms_sms.py

```diff
@@ -100,7 +100,7 @@ def _send(self, delete_all=False, raise_exception=False):
     def _postprocess_sent_sms(self, iap_results, failure_reason=None, delete_all=False):
         sms_to_notif_status = {
             'success': False, 'insufficient_credit': 'sms_credit',
-            'wrong_format_number': 'sms_number_format', 'server_error': 'sms_server'}
+            'wrong_number_format': 'sms_number_format', 'server_error': 'sms_server'}
```

Comment:
Hello,

https://github.com/odoo/iap-apps/blob/master/iap_services/iap_proxy_sms/controllers/main.py tells the inverse and that wrong_format_number is returned.

Reply:
There is and inconsistency between the doc of the route and the actual result returned. 
e.g.

https://github.com/odoo/iap-apps/blob/master/iap_services/iap_proxy_sms/controllers/main.py#L125
https://github.com/odoo/iap-apps/blob/master/iap_services/iap_proxy_sms/controllers/main.py#L162

We could make the change server-side but I think that 'wrong_number_format' suits better.

Reply:
I propose that you update the doc then :D .

Reply:
I will propagate wrong_number_format everywhere in code of odoo.

Reply:
on it :p